diffrax/term.py

```diff
@@ -413,6 +426,10 @@ def is_vf_expensive(
         _t1 = jnp.where(self.direction == 1, t1, -t0)
         return self.term.is_vf_expensive(_t0, _t1, y, args)
 
+    def levy_contr(self, t0: Scalar, t1: Scalar) -> (PyTree, PyTree):
+        assert isinstance(self.term, _ControlTerm)
+        return self.term.levy_contr(t0, t1)
```

Comment:
I think I'd suggest removing these extra methods and instead just have `contr` pipe any `**kwargs` forward.

Reply:
I wanted to make it explicit that a given method will always return LevyVal. But I suppose we can get rid of that. I'll just add the following check to all solvers requiring STLA:
```levy = diffusion.contr(t0, t1, use_levy=True)
        assert isinstance(levy, LevyVal) and (levy.H is not None), \
            ("The diffusion should be a ControlTerm controlled by either a"
             "VirtualBrownianTree or an UnsafeBrownianPath with"
             "`spacetime_levyarea` set to True.")
```

Reply:
Hmm I thought I removed this...